content/markdown/02-working-with-data/02-01-reading-writing.rmd

```diff
@@ -0,0 +1,147 @@
+---
+output:
+  pdf_document: default
+  html_document: default
+---
+# Reading and Writing Files in R 
+**Author:** Cole Brookson
+**Date:** 13 June 2022
+
+Quickly after beginning to use R, as a biologist it is likely you will want to analyze some data. These data will be often stored as external files in your computer, with the most common data file types being Excel Spreadsheet files (`.xls(x)`), Comma Separated Value files (`.csv`), or Plain Text (`.txt`) files.
+
+Since it is best practice to **never** directly edit your raw data files (because what if you need to go back to the original version!?), a common workflow is to:
+
+1. *Read your data into R *`
+
+2. Perform some kind of analysis, creating new data objects, figures, or outputs, then 
+
+3. *Write out those data objects/figures/outputs to your computer for later use*
+
+Steps 1 & 3 will be covered here. 
+
+There are, as there are with all things programming, multiple ways to perform these tasks in both base R and via packages. 
+
+## Reading Files 
+
+When reading in a new file, after manually checking it and ensuring it is formatted in a readable way, note the path in your computer where the file is present. **NOTE** see section on workflow and the `here()` package. 
```

Comment:
This is likely already on your mind, but we should hyperlink to the workflow section here.